third_party/move/move-compiler-v2/tests/checking/typing/borrow_field.exp

```diff
@@ -6,7 +6,5 @@ module 0x8675309::M {
     private fun t0(s: &M::S,s_mut: &mut M::S,s_mut2: &mut M::S): (&u64, &u64, &mut u64) {
         Tuple(Borrow(Immutable)(select M::S.f(s)), Borrow(Immutable)(select M::S.f(s_mut)), Borrow(Mutable)(select M::S.f(s_mut2)))
     }
-    spec fun $t0(s: M::S,s_mut: M::S,s_mut2: M::S): (&u64, &u64, &mut u64) {
```

Comment:
Could you explain why this output is changed?

Reply:
Good catch, thought was that fixed after all prover tests pass, but apparently not. Fixed.